core/src/impl/Kokkos_Atomic_Windows.hpp

```diff
@@ -52,8 +52,6 @@
 #include <winsock2.h>
 #include <windows.h>
 
-#undef VOID
```

Comment:
I believe this was a previous attempt to deal with the fact that winsock2 or so defines a VOID pragma ...